dev-docs/RFCs/v6.1/contour-layer-rfc.md

```diff
@@ -0,0 +1,75 @@
+# RFC: Contour Layer
+
+* **Authors**: Ravi Akkenapally
+* **Date**: June 2018
+* **Status**: For Review.
+
+## Overview
+
+This RFC proposes a new layer for rendering contours lines.
+
+
+## Use Cases
+
+Kepler and Manifold applications.
+
+
+## Proposed
+
+A new layer with following key props.
+
+* contours (Array): An array with each element containing a threshold and color values. Each element results in at most one contour with specified color.
+
+* cellSize (Number): Cell size in meters, used for aggregating data.
```

Comment:
in meters? can we specify screen pixel values?

Reply:
yes do able for screen space layer. To be precise, this in the same space where aggregation happens, if aggregating in lng/lat it will be in meters (converted to lng/lat offsets), if aggregating in screen space, it will be in pixels. Will update once screen space version is ready.